handler/core/strategy.go

```diff
@@ -0,0 +1,22 @@
+package core
```

Comment:
the core package (basically all oauth2 things) now defines what the different strategies must implement. this is an upside because if we have a "supersecure" package, it could use it's own strategy signatures